web/app.js

```diff
@@ -2656,69 +2658,43 @@ function onTouchMove(evt) {
   if (touch0.identifier > touch1.identifier) {
     [touch0, touch1] = [touch1, touch0];
   }
-  const { pageX: page0X, pageY: page0Y } = touch0;
-  const { pageX: page1X, pageY: page1Y } = touch1;
+  const { screenX: screen0X, screenY: screen0Y } = touch0;
+  const { screenX: screen1X, screenY: screen1Y } = touch1;
```

Comment:
Given the use of screenX/Y here now, is it expected to still use pageX/Y in the `onTouchStart` function above?
Note https://github.com/mozilla/pdf.js/pull/19124/files#diff-e24e4bd32090b0c400a70f02d1b844df95c9bbe62cd0b280f4397e9f4194fe44R2643-R2648

Reply:
Yes of course.

Reply:
In the latest version of the patch this is no longer fixed, was that intentional?

If so, please add a short comment explaining why the `onTouchStart` respectively `onTouchMove` function don't use the same property.